src/main/java/ru/bulldog/justmap/map/data/classic/ChunkDataManager.java

```diff
@@ -1,21 +1,20 @@
 package ru.bulldog.justmap.map.data.classic;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
The code base contains no `*` imports. I think this is a code style that is good to keep.

Reply:
No problem.  Looks like Intellij wil automatically replace the imports with a wildcard once more than five from the same package are specified.  * has been removed.